conanfile.py

```diff
@@ -11,45 +11,53 @@ class GTestConan(ConanFile):
     description = "Google's C++ test framework"
     url = "http://github.com/bincrafters/conan-gtest"
     license = "BSD 3-Clause"
-    exports = ["LICENSE.md"]
-    exports_sources = ["CMakeLists.txt"]
+    exports = ["LICENSE.md", "FindGTest.cmake", "FindGMock.cmake"]
```

Comment:
Should move these to export_sources

Reply:
Indeed! Done in the latest commit (just now).